### DISCUSSION TITLE
Migrating from Slack to GitHub Discussions

### DISCUSSION THREAD

**Discussion by kgilmer** [regolith-linux/regolith-desktop#668] [Announcements]
Starting with Regolith 2.1, we'll be updating links to point to this GitHub discussions app rather than Slack.  This is due to the following reasons:
* Some contributors and users have mentioned a dislike for Slack, problems with the client, etc.
* The free Slack account only provides a window of past data.  We lose our history.
* Work done in slack does not integrate with other items in GitHub such as issues, PRs, and releases, but GH discussions does.
* Learning about how other communities have made the same move and worked well for them
* Less is more

The Slack instance will remain as-is but the website will not reference it on the website.  Anyone with opinions on the move are welcome to express them here, your feedback is valued.

**Comment by Whyglobaleyes** [comment]
regolith was the only slack channel I still used - not sad to see you move.
